### ISSUE TITLE
Symbol for guard_rail is misleading

### ISSUE THREAD

**Issue by Langlaeufer** [openstreetmap/iD#9594]
### URL

https://www.openstreetmap.org/edit?editor=id&#map=21/51.82203/10.44329

### How to reproduce the issue?

https://wiki.openstreetmap.org/wiki/Tag:barrier%3Dguard_rail

Guard rails along roadways typically have an inner and outer surface; the inner surface is smooth and faces the roadway, while the outer surface is typically where vertical struts are connected (see example picture in info-box). If there is a clear inner/outer demarcation to the guard rail, construct the line so that the right side is inner and left side is outer.

ID shows the smooth side on the left and the arrows (which looks like the posts for me) on the right 

### Screenshot(s) or anything else?

![image](https://user-images.githubusercontent.com/15994341/233849219-161957c2-1184-4fc7-bc9e-7ccb4e1be48e.png)

### Which deployed environments do you see the issue in?

Released version at openstreetmap.org/edit

### What version numbers does this issue effect?

current (since introduction of this symbol?)

### Which browsers are you seeing this problem on?

Chrome (and all others)

**Comment by bhousel** [comment]
The convention in OSM is for all directional ways (like cliffs, retaining walls, coastlines, kerbs, and these guard rails) considers the "downward" direction to the right of the line, that's why the arrows point to the right.

**Comment by Langlaeufer** [comment]
Yes, in cartography it is convention for all embankment and cliff like structures that the arrows show downwards.
This make still sense at kerbs - because they are like small cliffs.

But this does not make sense for guard rails - they have no down side. Here I would expect an ground plan-like signature (view from top - where I can see the smooth side.

I found a lot streets with rail guards systematically in the wrong direction. So I guess I'm not the only person who do not understand the signature. Don't forget that not only the ID developers are using the editor. How would a ID novice understand the signature?

**Comment by Langlaeufer** [comment]
Instead of using arrows on the inside, how about using small circles (for the posts) on the outside?

**Comment by Langlaeufer** [comment]
https://community.openstreetmap.org/t/leitplanken-die-wiki-definition-der-seite-einer-leitplanke-ist-nicht-intuitiv/98444

**Comment by tyrasd** [comment]
2e3eda419c585ddd68b8302bfa346eda35885289 has an example of how it could look like with small circles on the outside of the barrier. For comparison there are `barrier=wall` and `man_made=embankement` included in the screenshot:

<img width="400" src="https://github.com/user-attachments/assets/1ab1fe6f-edfd-45e0-876e-ce51ff249885" />

Feedback for this proposal would be appreciated.

**Comment by Dimitar5555** [comment]
Looks great! Does it handle double sided guard rails? I can't recall what tag was used for them atm.

**Comment by Langlaeufer** [comment]
[two_sided=yes](https://wiki.openstreetmap.org/wiki/Key%3Atwo_sided)

**Comment by tordans** [comment]
Do we actually need an indicator for the "down side" for guard rails? We don't have them for `barrier=fence+fence_type=railing`

> <img width="464" height="480" alt="Image" src="https://github.com/user-attachments/assets/5f7be303-ab53-4957-9560-3d71d11f1571" />

I think it would be enough to just re-use this style for the guard rail. It has a similar function and will "never" be show up in the same place, so its unlikely to mix them up.

**Comment by Dimitar5555** [comment]
> Do we actually need an indicator for the "down side" for guard rails? We don't have them for `barrier=fence+fence_type=railing`

Railings don't have any "sideness". They are the same on both sides. See https://wiki.openstreetmap.org/wiki/Tag:fence_type%3Drailing.

**Comment by tyrasd** [comment]
> Do we actually need an indicator for the "down side" for guard rails?

I would have said that any tag where the wiki recommend a particular drawing direction should somehow indicate the sidedness of the way.

> `two_sided=yes`

Good point: in this case iD currently just skips the direction markers and draws it like a regular barrier. That should be fine, I think.

**Comment by matkoniecz** [comment]
> Feedback for this proposal would be appreciated.

it works perfectly well for me
